frontend/ios/Runner.xcodeproj/xcshareddata/xcschemes/Bayern.xcscheme

```diff
@@ -10,7 +10,7 @@
             ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
             <ActionContent
                title = "Run Script"
-               scriptText = "source $HOME/.bashrc&#10;cd &quot;$SRCROOT&quot; &amp;&amp; app-toolbelt v0 build-config write-xcconfig bayern ios --directory &quot;$SRCROOT&quot;&#10;"
+               scriptText = "source $HOME/.bashrc&#10;cd &quot;$SRCROOT&quot; &amp;&amp; app-toolbelt v0 build-config write-xcconfig bayern ios --directory &quot;$SRCROOT&quot; &amp;&amp; fvm flutter pub run build_runner build --define &quot;df_build_config=name=bayern&quot;&#10;"
```

Comment:
Not sure if we want this. This changes the build config when starting a build through xcode